src/Blazored.LocalStorage/Serialization/SystemTextJsonSerializer.cs

```diff
@@ -1,14 +1,8 @@
-﻿using Blazored.LocalStorage.StorageOptions;
+using Blazored.LocalStorage.StorageOptions;
 using Microsoft.Extensions.Options;
 using System.Runtime.CompilerServices;
 using System.Text.Json;
 
-[assembly: InternalsVisibleTo("Blazored.LocalStorage.Tests, PublicKey=" +
-"0024000004800000940000000602000000240000525341310004000001000100e94102d6760ebc" +
-"ff1970798791888ddf102ac709e19db9a312721fafca42b894652b59bada7d592a4ab62a5b7650" +
-"7a27720e922bc310c4f5aa75acd8ab59632c920ac41a7e9abcaf4b8bb5525a60931faccea704db" +
-"dcf68e1207616751447dcfec687f18854148aa66a9a09e1edc8fd0c9bd11950b4baf7d46fe3899" +
-"3af4add4")]
```

Comment:
Why did you choose to remove this attribute and move it to the csproj file?

Reply:
The presence of `[InternalsVisibleTo]` will auto-generate a AssemblyInfo.cs file at compile time no matter where it lives. By having it appear in the `SystemTextJsonSerializer.cs` file may hide that the fact that there are other arguably more important internals that we want visible to friend assemblies (e.g. `IStorageProvider`). Having this in the csproj gives it a centralized location. This is my own opinion of course, and I can happily revert. :smile: